README.md

```diff
@@ -96,7 +96,7 @@ Using Docker Compose is a convenient way to deploy and manage Nebula Graph.
    * For Nebula Graph 2.0:
 
     ```shell
-    $ docker run --rm -ti --network nebula-docker-compose_nebula-net --entrypoint=/bin/sh vesoft/nebula-console:v2-preview-nightly
+    $ docker run --rm -ti --network nebula-docker-compose_nebula-net --entrypoint=/bin/sh vesoft/nebula-console:v2-nightly
```

Comment:
It seems like there is no such a tag on docker hub: https://hub.docker.com/r/vesoft/nebula-console/tags?page=1&ordering=last_updated.
Am I missing anything?

Reply:
It's changed just now, and tomorrow there will be an image named nebula-console:v2-nightly

Reply:
Cool